civic-tech-club/Dockerfile

```diff
@@ -1,20 +1,35 @@
-# Use the official Node.js image as the base image
-FROM node:22-alpine
+# Use an official Python runtime as a parent image
+FROM python:3.9-slim
 
-# Set the working directory inside the container
+# Set the working directory in the container
 WORKDIR /app
 
-# Copy package.json and package-lock.json to the working directory
-COPY package*.json ./
+# Install nodejs needed for the Frontend
```

Comment:
It would be more deterministic to use a separate image for nodejs here and copy the binaries into the Python container. That allows us to pin official image versions with reliable build processes and docker pulls instead of relying on package registries where we will end up installing the latest versions of these tools every time we build or deploy.